spec/graphql/relay/page_info_spec.rb

```diff
@@ -75,8 +75,8 @@ def get_last_cursor(result)
       result = star_wars_query(query_string, "last" => 1, "first" => 1, "before" => cursor_of_last_base)
       assert_equal(true, get_page_info(result)["hasNextPage"])
       assert_equal(true, get_page_info(result)["hasPreviousPage"])
-      assert_equal("Mg==", get_page_info(result)["startCursor"])
-      assert_equal("Mg==", get_page_info(result)["endCursor"])
+      assert_equal("MQ==", get_page_info(result)["startCursor"])
+      assert_equal("MQ==", get_page_info(result)["endCursor"])
```

Comment:
:+1: Confirmed the changed offset in the test matches the bases you named! 

![image](https://cloud.githubusercontent.com/assets/2231765/26463738/520c7c3e-4153-11e7-88b3-3df896d7d09a.png)